### ISSUE TITLE
put out beta of v4 with access tokens

### ISSUE THREAD

**Issue by incanus** [mapbox/DEPRECATED-mapbox-ios-sdk#468]
Support `v4` of the REST API with access tokens per https://www.mapbox.com/developers/api/#access-tokens. Should just need to hit `RMMapboxSource`.

**Comment by incanus** [comment]
Auto-uses `v4` API for TileJSON, tiles, markers, and grids if you do the following anyplace: 

``` objc
[[RMConfiguration configuration] setAccessToken:@"<token>"];
```

**Comment by willwhite** [comment]
This was [reverted in v1.4.0](https://github.com/mapbox/mapbox-ios-sdk/blob/release/CHANGELOG.md#140). Need to take another pass.

**Comment by incanus** [comment]
This is out in beta, in the develop branch currently. 

## 

Justin R. Miller
justin@mapbox.com
@incanus77

**Comment by willwhite** [comment]
Ah ok, so we just need to roll a release?

**Comment by incanus** [comment]
Yeah, it's mostly related to testing of #132, which worked around an iOS bug and as a result also worked around the problem with 512px tiles. Will roll this week.

**Comment by incanus** [comment]
Yeah, just need to release to bring in work from https://github.com/mapbox/mapbox-ios-sdk/issues/494.

**Comment by incanus** [comment]
SDK `1.5.0` is now released, which includes this and #494.
